homeassistant/components/imeon_inverter/strings.json

```
@@ -0,0 +1,179 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Add Imeon Inverter",
+        "data": {
+          "address": "[%key:common::config_flow::data::url%]",
+          "username": "[%key:common::config_flow::data::username%]",
+          "password": "[%key:common::config_flow::data::password%]"
+        },
+        "data_description": {
+          "address": "The address of the inverter",
+          "username": "Your inverter login",
+          "password": "Your inverter password"
+        }
+      }
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
+    }
+  },
+  "entity": {
+    "sensor": {
+      "battery_autonomy": {
+        "name": "Battery Autonomy"
+      },
+      "battery_charge_time": {
+        "name": "Battery Charge Time"
+      },
+      "battery_power": {
+        "name": "Battery Power"
+      },
+      "battery_soc": {
+        "name": "Battery SOC"
+      },
+      "battery_stored": {
+        "name": "Battery Stored"
```

```suggestion
        "name": "Battery stored"
```
for the rest as well (use sentence case)